src/styles/main.scss

```diff
@@ -1,3 +1,233 @@
-body {
+@mixin reset {
   margin: 0;
+  padding: 0;
+}
+
+@mixin justify-space-between {
+  display: flex;
+  justify-content: space-between;
+}
+
+@mixin justify-center {
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+@mixin font-weight-size-height (
+  $font-weight: 400,
+  $font-size: 12px,
+  $line-height: 18px
+) {
+  font-weight: $font-weight;
+  font-size: $font-size;
+  line-height: $line-height;
+}
+
+@mixin square ($square-width) {
+  width: $square-width;
+  height: $square-width;
+}
+
+@function calculateMaxWidth($content-width, $content-gap, $quantity: $columns) {
+  @return $quantity * $content-width + ($quantity - 1) * $content-gap;
+}
+
+$font-roboto: roboto, helvetica, sans-serif;
```

Comment:
it's a bit useless to create variable for font. add this to `html` instead, it will be enough. name font from upper case, `Roboto`